dwitter/static/main.css

```diff
@@ -274,13 +274,11 @@ iframe {
 }
 
 .head-account-info .collapsible {
-  opacity: 0;
   position: absolute;
   border: 1px solid #ddd;
   right: 1em;
-  margin-top: -4px;
-  transition: all 0.05s;
```

Comment:
I had to be a little sneaky with transitions to get the opacity back in (since you can't transition display none/block), but it worked \o/